app/views/admin/projects.html.erb

```diff
@@ -73,52 +48,107 @@ See doc/COPYRIGHT.rdoc for more details.
           <label for='name'><%= Project.model_name.human %>:</label>
           <%= text_field_tag 'name', params[:name] %>
       </li>
+      <li class="simple-filters--filter">
+        <label for='' aria-visible="false" style="visibility: hidden"><%= l(:button_apply) %></label>
```

Comment:
It's another hack to align the button with the inputs as *demanded* to be in accordance with the visual. We will need to alter the `simple-filters` namespace to respect the visual style. This will not make the cut before the demo, though, and this makes me introduce these lovely hacks to finish something in time ;)